README.md

```diff
@@ -80,7 +80,7 @@ python -m venv venv
 
 3. Activate the virtual environment
    - For macOS and linux, run `source venv/bin/activate`
-   - For windows, run `venv\Scripts\activate.bat`
+   - For windows, run `.\env\Scripts\activate`
```

Comment:
In the readme, the name of the virtual environment is `venv`, therefore this command should be `.\venv\Scripts\activate`.

Reply:
Ok yeah ok